client/src/hm2.legend.js

```diff
@@ -1,160 +1,180 @@
-import {select} from 'd3-selection'
+import { select } from 'd3-selection'
```

Comment:
i don't think this file is used at all?

Reply:
Thanks, just deleted the new CNV categories added earlier. Rest are just prettier format changes